### PR TITLE
XGAL-72 User Activity Notification

### PR DESCRIPTION
**Pull request by thongphp**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -32,12 +32,12 @@ REDIS_PORT=6379
 
 MAIL_DRIVER=smtp
 MAIL_MAILER=smtp
-MAIL_HOST=smtp.gmail.com
-MAIL_PORT=587
-MAIL_USERNAME=
-MAIL_PASSWORD=
-MAIL_ENCRYPTION=tls
-MAIL_FROM_ADDRESS=
+MAIL_HOST=localhost
```
Comment (jooservices) [inline .env.example:35]: Do not change these lines. By default i think it should be pointed to gmail instead "nothing"

Comment (jooservices) [reply .env.example:35]: Please revert back to default with my email soulevilx@gmail.com and all related. Except password

**Review by jooservices — CHANGES_REQUESTED** [review]

File: phpunit.xml [hunk 2]
```diff
@@ -1,29 +1,26 @@
...
-        <server name="BCRYPT_ROUNDS" value="4"/>
-        <server name="CACHE_DRIVER" value="array"/>
-        <server name="DB_CONNECTION" value="sqlite"/>
-        <server name="DB_DATABASE" value=":memory:"/>
-        <server name="MAIL_MAILER" value="array"/>
-        <server name="QUEUE_CONNECTION" value="sync"/>
-        <server name="SESSION_DRIVER" value="array"/>
-        <server name="MONGO_DB_DATABASE" value="xgallery_test" />
-    </php>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd" bootstrap="vendor/autoload.php" colors="true">
```
Comment (jooservices) [inline phpunit.xml:2]: Why are you changing this ?

Comment (thongphp) [reply phpunit.xml:2]: According that PHPUnit has been update to version 9.x there are some things change in XML file.

Also the Scrutinizer can not build and run tests too.

After I use phpunit command option for update this file.

Now Scrutinizer works again.

**Pull request merged**